save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesSourceController.kt

```diff
@@ -605,12 +605,6 @@ class TestSuitesSourceController(
                 .contentType(MediaType.APPLICATION_OCTET_STREAM)
                 .body(testSuitesSourceSnapshotStorage.download(it))
         }
-        .onErrorReturn(
```

Comment:
Checks for existence of keys are performed earlier and converted to `NOT_FOUND` there. This `onErrorReturn` basically was wrapping `ResponseStatusException`s with `404`s into another `404`, and message was being lost.

I believe if there is an error in `testSuitesSourceSnapshotStorage.download` after several existence checks, then it's okay to return a `500` error

cc @nulls